WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -567,6 +567,9 @@
 /* Alert option to add document contents into a blog post. */
 "Add Contents to Post" = "Add Contents to Post";
 
+/* No comment provided by engineer. */
+"Add description" = "Add description";
```

Comment:
There are quite a few strings without reverse-DNS key in this diff. All but [one](https://github.com/wordpress-mobile/WordPress-iOS/commit/1c40a9b2123af8d2bfd9490d66e233cc06c3737f#r109225555) come from libraries (Gutenberg, would be my guess). There's not much we can do about it right now, but it gave me the idea of possibly updating the automation that extracts the strings to generate the reverse DNS key based on the library name and the value. 

E.g.:

```
"pods.gutenberg.add_description" = "Add description";
```

Reply:
Yep, this string comes from the Jetpack's VideoPress block ([reference](https://github.com/Automattic/jetpack/blob/fd90454188d21d1bbcf16ff50dbdb74803729ccb/projects/packages/videopress/src/client/block-editor/blocks/video/components/details-panel/index.native.js#L29)).

Reply:
Thanks for confirming @fluiddot 🙇‍♂️